packages/plugin/src/generators/executor/executor.ts

```diff
@@ -16,6 +16,7 @@ import type { Schema } from './schema';
 import * as path from 'path';
 import { PackageJson } from 'nx/src/utils/package-json';
 import pluginLintCheckGenerator from '../lint-checks/generator';
+import { nxVersion } from '@nx/js/src/utils/versions';
```

Comment:
I think this is coming from the wrong place. Is there no `nxVersion` here in this package? We can add one.

Reply:
I will add one then.